benchmark/README.md

```diff
@@ -15,3 +37,26 @@ bash scripts/configure-environment.sh
 ```
 
 * Follow the guidelines until installation is complete. 
+
+## Configure the inputs
+
+* You can configure following options in `/benchmark/config.json`
+
+| Option | Description | Default Value |
+| --- | --- | --- |
+| types | List of databases to test | cassandra,mysql,elasticsearch|
+| outputFile | Name of the result's file | index.html |
+| scaleFactors | List of scale factors for dataset (1.0 indicates 1 GB) | 1.0 |
```

Comment:
Is there a minimum range for the scale factor? Is it integer or fractional values?

Reply:
there is a max limit. added.